src/backend/db/products.js

```diff
@@ -1,30 +1,36 @@
-import { v4 as uuid } from "uuid";
+import { v4 as uuid } from 'uuid';
 
 /**
  * Product Database can be added here.
  * You can add products of your wish with different attributes
  * */
-
+import { realMadrid, football, manchesterUnited } from './../../assets/index';
 export const products = [
   {
     _id: uuid(),
-    title: "You Can WIN",
-    author: "Shiv Khera",
-    price: "5000",
-    categoryName: "non-fiction",
+    title: 'Real Madrid Jersey',
+    price: '5000',
+    categoryName: 'jersey',
+    inWishlist: true,
+    imgSrc: realMadrid,
+    isBestSeller: true,
   },
   {
     _id: uuid(),
-    title: "You are Winner",
-    author: "Junaid Qureshi",
-    price: "3000",
-    categoryName: "horror",
+    title: 'Nike Strike Football',
+    price: '3000',
+    inWishlist: false,
+    categoryName: 'football',
+    inWishlist: false,
+    imgSrc: football,
   },
   {
     _id: uuid(),
-    title: "Think and Grow Rich",
-    author: "Shiv Khera",
-    price: "1000",
-    categoryName: "fiction",
+    title: 'Manchecter United Jersey',
+    price: '5000',
+    inWishlist: true,
+    categoryName: 'jersey',
+    inWishlist: false,
+    imgSrc: manchesterUnited,
   },
```

Comment:
For the products array, you are using `inWishlist` key twice for last 2 objects. 
Also I would suggest keep the order of keys for each object same, it will ease in reading